test/one/nio/http/HttpMethodTest.java

```diff
@@ -34,15 +35,19 @@
  * @author Vadim Tsesko <mail@incubos.org>
  */
 public class HttpMethodTest {
-    private static final String URL = "http://127.0.0.1:8181";
+    private static String URL = "http://127.0.0.1";
     private static final String ENDPOINT = "/echoMethod";
 
     private static HttpServer server;
     private static HttpClient client;
 
     @BeforeClass
     public static void beforeAll() throws IOException {
-        server = new TestServer(HttpServerConfigFactory.create(8181));
+        ServerSocket dummySocket = new ServerSocket(0);
+        int availablePort = dummySocket.getLocalPort();
+        dummySocket.close();
+        server = new TestServer(HttpServerConfigFactory.create(availablePort));
+        URL += ":" + availablePort;
```

Comment:
There is no need to update URL as it is not used anywhere exception this method.

Reply:
Good idea.